src/events/messageUpdate.ts

```diff
@@ -5,6 +5,8 @@ export default class extends Event {
     constructor(client: KlasaClient, store: EventStore, file: string[], dir: string) {
         super(client, store, file, dir, {
             enabled: true,
+            name: 'janetMessageUpdate',
```

Comment:
Is the name property really required since it defaults to filename?

Reply:
Yes. But the issue is it's overriding the built in event of klasa https://github.com/dirigeants/klasa/blob/master/src/events/messageUpdate.js
This was fixed in sgn by adding `core` to the `name` of the files but it hasn't reached master yet.